ParseUI.framework/Headers/PFCollectionViewCell.h

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Anything from this file and below is auto generated code added when I included the parseUI framework. Ignore this